servicex/servicex_client.py

```diff
@@ -214,15 +221,38 @@ def _output_handler(
 
 
 def deliver(
-    config: Union[ServiceXSpec, Mapping[str, Any], str, Path],
+    spec: Union[ServiceXSpec, Mapping[str, Any], str, Path],
```

Comment:
Is this a breaking change. ;-) But I think `spec` is way better!

Reply:
It is a breaking change if anyone is passing `config=` as a keyword argument, yeah... none of our examples do this though. Should definitely warn people about it when the release is made.